tests/test_buffer.py

```diff
@@ -462,5 +462,133 @@ def test_SimpleArray_broadcast_ellipsis_dtype(self):
                     self.assertEqual(v, sarr[i, j, k])
                     v += 1
 
+    def test_SimpleArray_broadcast_slice_basic(self):
+        ndarr_input = np.arange(
+            1*2*3*4*5, dtype='float64').reshape((1, 2, 3, 4, 5))
+        ndarr_input += 999  # just a magic number
+
+        def init(sarr):
+            v = 0
+            for x1 in range(sarr.shape[0]):
+                for x2 in range(sarr.shape[1]):
+                    for x3 in range(sarr.shape[2]):
+                        for x4 in range(sarr.shape[3]):
+                            for x5 in range(sarr.shape[4]):
```

Comment:
This is a good place for some humourous comments :)

Reply:
could you provide some examples(?

Reply:
That won't be humourous anymore.  Let's try it next time :)